### PR TITLE
feature-mathMagicianComponents

### PR DESCRIPTION
**Pull request by maths29**
in this project I:
- Refactored the components
- components.js
- app.css,
- index.js
- I kindly request you to review the code and provide your valuable feedback. Feel free to comment directly on the pull request or raise any concerns you may have. Your input will greatly contribute to enhancing the quality and functionality of the project.

Thank you for taking the time to review my code. I'm looking forward to your feedback!

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <div className="button-div">
+      <button type="button" className="zero">0</button>
+      <button type="button">.</button>
+      <button type="button" className="operation">=</button>
+    </div>
+
+  </div>
+);
+
+export default Calculator;
```
Comment (muneebulrehman) [inline src/components/calculator.js]: The layout of the calculator should match that of the provided design. The barebone structure should be the same. But as you can see, your layout is breaking. Kindly fix this issue.

| Your design  | Provided design |
| ------------- | ------------- |
| image | image |

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+const Calculator = () => (
```
Comment (muneebulrehman) [inline src/components/calculator.js:4]: The `calculator.js` component should have at least 1 child component. I see you have created a `Button` component but are not using it anywhere. Kindly use at least 1 child component inside the `Calculator.js` be it the `Button.js` component.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @maths29 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job setting up the project. 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
Hi @maths29 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All the boilerplate that got created by the CRA is deleted. 👍🏽
* The project has a directory called `components`. 💯
* `Calculator.js` file is inside the components directory. 👌🏽
* `Calculator.js` component is a **functional** component 💪🏽

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**